backend/twitterbot/management/commands/load_tweetinfo.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path
+
+from django.core.management.base import BaseCommand
+
+from tournesol.entities.video import YOUTUBE_UID_NAMESPACE
+from tournesol.models.entity import Entity
+from twitterbot.models.tweeted import TweetInfo
+
+
+class Command(BaseCommand):
+    help = "Load already tweeted videos from a file and save data as TweetInfo."
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "-f",
+            "--file",
+            type=str,
+            help="Relative path to the input file.",
+        )
```

Comment:
Running the command fails when the -f option is not provided. We should mark the -f argument as required to make the command display a nice error message.

```suggestion
            required=True,
        )
```